### PR TITLE
add a M1 blog post

### PR DESCRIPTION
**Pull request by mattip**
A short blog post about the M1 work

### CODE REVIEW

**Comment by cfbolz** [comment]
maybe we can link to the aarch64 blog post? https://www.pypy.org/posts/2019/07/pypy-jit-for-aarch64-7161523403247118006.html

should we ask andrew whether he wants to be mentioned?

**Comment by mattip** [comment]
> maybe we can link to the aarch64 blog post? https://www.pypy.org/posts/2019/07/pypy-jit-for-aarch64-7161523403247118006.html

OK

> ... whether he wants to be mentioned?

The large donation was made anonymously.

File: posts/2022/07/m1-support-for-pypy.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.. title: M1 support for PyPy
+.. slug: m1-support-for-pypy
+.. date: 2022-07-21 18:27:14 UTC
+.. tags: release
```
Comment (cfbolz) [inline posts/2022/07/m1-support-for-pypy.txt:4]:
```suggestion
.. tags:
```

[the remark this answers is not in the document]
Comment (mattip) [reply posts/2022/07/m1-support-for-pypy.txt]: done, thanks

**Pull request merged**